R/calculateUnifrac.R

```diff
@@ -104,30 +85,28 @@
 #' library(scater)
 #' calculateUnifrac(esophagus, weighted = FALSE)
 #' calculateUnifrac(esophagus, weighted = TRUE)
-#' calculateUnifrac(esophagus, weighted = TRUE, normalized = FALSE)
 #' # for using calculateUnifrac in conjunction with runMDS the tree argument
 #' # has to be given separately. In addition, subsetting using ntop must
 #' # be disabled
 #' esophagus <- runMDS(esophagus, FUN = calculateUnifrac, name = "Unifrac",
 #'                     tree = rowTree(esophagus),
-#'                     exprs_values = "counts",
+#'                     assay.type = "counts",
 #'                     ntop = nrow(esophagus))
 #' reducedDim(esophagus)
 NULL
 
 #' @rdname calculateUnifrac
 #' @export
 setGeneric("calculateUnifrac", signature = c("x", "tree"),
-           function(x, tree, ... )
-             standardGeneric("calculateUnifrac"))
+            function(x, tree, ... )
+                standardGeneric("calculateUnifrac"))
 
 #' @rdname calculateUnifrac
 #' @export
 setMethod("calculateUnifrac", signature = c(x = "ANY", tree = "phylo"),
-    function(x, tree, weighted = FALSE, normalized = TRUE,
-             BPPARAM = SerialParam(), ...){
+    function(x, tree, weighted = FALSE, normalized = TRUE, ...){
```

Comment:
normalized should be removed. We do not have that option anymore. Also from TreeSE method